scripts/build.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+
+# http://redsymbol.net/articles/unofficial-bash-strict-mode/
+IFS=$'\n\t'
+set -euxo pipefail
+
+if [ ! -f ./config/private.yml ]; then
+  echo '{}' > ./config/private.yml
+  echo "you need to set up your private.yml"
+fi
+
+# setting up the local blobstore
+cp config/final.yml config/final.yml.bk
```

Comment:
This is just saving the final config as a backup instead of replacing it directly with the local config (in the next line). It's just for convenience.